numpy/core/src/multiarray/arraytypes.c.src

```diff
@@ -1849,7 +1849,57 @@ BOOL_fromstr(char *str, void *ip, char **endptr,
 }
 
 /**begin repeat
- * #fname = CFLOAT, CDOUBLE, CLONGDOUBLE,
+ * #fname = CFLOAT, CDOUBLE#
+ * #type = npy_cfloat, npy_cdouble#
+ */
+static int
+@fname@_fromstr(char *str, void *ip, char **endptr,
+        PyArray_Descr *NPY_UNUSED(ignore))
+{
+    double result;
+
+    result = NumPyOS_ascii_strtod(str, endptr);
+
+    @type@ output;
+
+    if (endptr && (*endptr[0] == '+') || (*endptr[0] == '-')) {
```

Comment:
The compiler is suggesting an extra set a parenthesis here for clarification, that is why the test failures.

Reply:
@zipoh ping

Reply:
@zjpoh ping

Reply:
Sorry. I'm not available until next Monday. I'll fix it next Monday.